tests/components/plugwise/test_select.py

```diff
@@ -77,3 +77,10 @@ async def test_adam_select_regulation_mode(
         "heating",
         "on",
     )
+
+
+async def test_legacy_anna_select_entities(
+    hass: HomeAssistant, mock_smile_adam: MagicMock, init_integration: MockConfigEntry
+) -> None:
+    """Test a legacy thermostat select."""
```

Comment:
I think this docstring can be a bit more descriptive, I now have to assume that we are testing that anna doesn't create select entities?

Reply:
Correct, see updated docstring.